api/client/client.go

```diff
@@ -2863,6 +2869,24 @@ func (c *Client) GetClusterAuditConfig(ctx context.Context) (types.ClusterAuditC
 	return resp, nil
 }
 
+// GetAutoupdateConfig gets autoupdate configuration.
+func (c *Client) GetAutoupdateConfig(ctx context.Context) (*autoupdate.AutoupdateConfig, error) {
+	resp, err := c.AutoupdateServiceClient().GetAutoupdateConfig(ctx, &autoupdate.GetAutoupdateConfigRequest{})
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+	return resp, nil
+}
+
+// GetAutoupdateVersion gets autoupdate version.
+func (c *Client) GetAutoupdateVersion(ctx context.Context) (*autoupdate.AutoupdateVersion, error) {
+	resp, err := c.AutoupdateServiceClient().GetAutoupdateVersion(ctx, &autoupdate.GetAutoupdateVersionRequest{})
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+	return resp, nil
+}
```

Comment:
after adding get methods for fetching config and version in `find` endpoint
```golang
interface ReadRemoteProxyAccessPoint {
	GetAutoupdateConfig(ctx context.Context) (*autoupdate.AutoupdateConfig, error)
	GetAutoupdateVersion(ctx context.Context) (*autoupdate.AutoupdateVersion, error)
}
```
`cache.Cache` and `client.ClientI` should implement these methods, otherwise if I replace this one to client get method I need to add some wrapper for https://github.com/gravitational/teleport/pull/45617/files#diff-3e237f973a30f7092dc18a706ecd95d8a7e1d99c5c50626b8bb0e7ac6cb7efd1R1899-R1920